subschemas/substrate-chain/src/resolvers/Query/democracy.ts

```diff
@@ -117,40 +112,37 @@ function formatReferendumData(
   bestNumber: BlockNumber,
 ): DemocracyReferendum | null {
   const imageProposal = referendum.image?.proposal;
-  if (imageProposal) {
-    const remainBlock = bestNumber ? referendum.status.end.sub(bestNumber).isub(BN_ONE) : undefined;
-    const {timeStringParts: endPeriod} = getBlockTime(api, remainBlock);
-
-    const enactBlock = bestNumber ? referendum?.status.end.add(referendum.status.delay).sub(bestNumber) : undefined;
-    const {timeStringParts: activatePeriod} = getBlockTime(api, enactBlock);
-
-    const ayePercent = !referendum.votedTotal.isZero()
-      ? referendum.allAye
-          .reduce((total: BN, {balance}) => total.add(balance), new BN(0))
-          .muln(10000)
-          .div(referendum.votedTotal)
-          .toNumber() / 100
-      : 0;
-
-    const meta = formatCallMeta(imageProposal.registry.findMetaCall(imageProposal.callIndex).meta);
-    return {
-      meta,
-      endPeriod,
-      activatePeriod,
-      votedAye: referendum.votedAye.toString(),
-      formattedVotedAye: formatBalance(api, referendum.votedAye),
-      votedNay: referendum.votedNay.toString(),
-      formattedVotedNay: formatBalance(api, referendum.votedNay),
-      voteCountAye: referendum.voteCountAye.toString(),
-      voteCountNay: referendum.voteCountNay.toString(),
-      ayePercent,
-      index: referendum.index.toString(),
-      hash: String(imageProposal.hash),
-      ...getCallParams(imageProposal),
-    };
-  }
-
-  return null;
+  const remainBlock = bestNumber ? referendum.status.end.sub(bestNumber).isub(BN_ONE) : undefined;
+  const {timeStringParts: endPeriod} = getBlockTime(api, remainBlock);
+
+  const enactBlock = bestNumber ? referendum?.status.end.add(referendum.status.delay).sub(bestNumber) : undefined;
+  const {timeStringParts: activatePeriod} = getBlockTime(api, enactBlock);
+
+  const ayePercent = !referendum.votedTotal.isZero()
+    ? referendum.allAye
+        .reduce((total: BN, {balance}) => total.add(balance), new BN(0))
+        .muln(10000)
+        .div(referendum.votedTotal)
+        .toNumber() / 100
+    : 0;
+
+  const meta = imageProposal ? formatCallMeta(imageProposal.registry.findMetaCall(imageProposal.callIndex).meta) : '';
+  let formatedReferendum = {
+    meta,
+    endPeriod,
+    activatePeriod,
+    votedAye: referendum.votedAye.toString(),
+    formattedVotedAye: formatBalance(api, referendum.votedAye),
+    votedNay: referendum.votedNay.toString(),
+    formattedVotedNay: formatBalance(api, referendum.votedNay),
+    voteCountAye: referendum.voteCountAye.toString(),
+    voteCountNay: referendum.voteCountNay.toString(),
+    ayePercent,
+    index: referendum.index.toString(),
+    hash: String(imageProposal?.hash),
+    ...(imageProposal ? getCallParams(imageProposal) : {}),
+  };
+  return formatedReferendum;
```

Comment:
Suggestion:

Simplify this part of the code! 

just do:
```suggestion
  return {
    meta,
    endPeriod,
    activatePeriod,
    votedAye: referendum.votedAye.toString(),
    formattedVotedAye: formatBalance(api, referendum.votedAye),
    votedNay: referendum.votedNay.toString(),
    formattedVotedNay: formatBalance(api, referendum.votedNay),
    voteCountAye: referendum.voteCountAye.toString(),
    voteCountNay: referendum.voteCountNay.toString(),
    ayePercent,
    index: referendum.index.toString(),
    hash: String(imageProposal?.hash),
    ...(imageProposal ? getCallParams(imageProposal) : {}),
  };
```



PS. I would avoid using `let` - using `const` is better if you're not changing it 😉

Reply:
Ohh, I assumed that i made the change Since I used it differently. So I'll make the change.